### ISSUE TITLE
`Error: Unsupported block type` with Terraform v0.12.6

### ISSUE THREAD

**Issue by nhooey** [BigDataBoutique/elasticsearch-cloud-deploy#76]
When running `terraform plan` from the `terraform-aws` directory, using Terraform v0.12.6, I get many of these errors:

```
Error: Unsupported block type

  on client.tf line 4, in data "template_file" "client_userdata_script":
   4:   vars {

Blocks of type "vars" are not expected here. Did you mean to define argument
"vars"? If so, use the equals sign to assign it a value.
```

It seems like a syntax incompatibility.

Any ideas?

**Comment by nhooey** [comment]
Fixed by changing `vars {` to `vars = {`, and similarly with `tags`.

**Comment by synhershko** [comment]
@nhooey thanks for reporting, @tsimonyan can you please have a look?

**Comment by bpluly** [comment]
You need to run the upgrade for terraform 0.12 on the module.

**Comment by synhershko** [comment]
Closed via #84
